x-pack/qa/operator-privileges-tests/src/main/java/org/elasticsearch/example/actions/GetActionsRequest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.example.actions;
+
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.common.io.stream.StreamInput;
+
+import java.io.IOException;
+
+public class GetActionsRequest extends ActionRequest {
```

Comment:
You can move this as an inner class of `GetActionsAction` (like autoscaling does) if you want.

Reply:
All transport stuff is not needed because its handled with reflection in RestAction.